soc/nxp/imx/imx8m/adsp/memory.h

```diff
@@ -78,20 +86,27 @@
 #define IDT_SIZE	0x2000
 
 /* physical DSP addresses */
-#define IRAM_BASE	0x3B6F8000
+/*#define IRAM_BASE	0x3B6F8000*/
+#define IRAM_BASE   0x60600000
 #define IRAM_SIZE	0x800
 
-#define DRAM0_BASE	0x3B6E8000
-#define DRAM0_SIZE	0x8000
+/*#define DRAM0_BASE	0x3B6E8000*/
+#define DRAM0_BASE  	0x70800000
+#define DRAM0_SIZE		0x8000
 
-#define DRAM1_BASE	0x3B6F0000
-#define DRAM1_SIZE	0x8000
+/*#define DRAM1_BASE	0x3B6F0000*/
+#define DRAM1_BASE  	0x80608000
```

Comment:
Same here, by the way was your intent to make this 0x8060 or was it 0x6060 for DRAM1?